fox_scraper/fox_scraper.py

```diff
@@ -38,6 +40,28 @@ def print_item(self):
         print "item name:", self.item_name
         print "item price: %.2f " %self.item_price
 
+    def print_assert(self):
+        return self.item_img_id + self.item_main_category + self.item_type + self.item_name + str(self.item_price)
+
+
+class FoxRabbitmq(object):
+    def __init__(self):
+        self.connection = pika.BlockingConnection(pika.ConnectionParameters(host='localhost'))
+        self.channel = self.connection.channel()
+        self.channel.exchange_declare(exchange='fox_scrap', type='direct')
+
+#    def establish_rabbitmq_connection(self):
+#        self.channel = self.connection.channel()
+#        self.channel.exchange_declare(exchange='fox_scrap', type='direct')
+
+    def close_rabbitmq_connection(self):
+        self.connection.close()
+
+    def send_message_to_rabbitmq(self, message):
+        message.print_item()
+        body = json.dump([m.__dict__ for m in message])
```

Comment:
message is an FoxItem class - it still not working.
but a little question - can I use other packages? (helper pkg...)